.gitignore

```diff
@@ -25,8 +25,6 @@ licenses
 **/.asciidoctor/images/*
 **/.asciidoctor/diagram/*
 **/images/diag-*.svg
-fineract-avro-schemas/out/
```

Comment:
Why did you remove these lines?

Reply:
I removed all those line to set a more general line for the other modules.  

`**/out`

Reply:
`fineract-provider/src/main/generated/` is not matching for `**/out`. Please fix it

Reply:
Also what is `gradleExp`? Please remove